tools/SampleAnalyzer/Commons/DataFormat/MCParticleFormat.h

```diff
@@ -131,6 +131,7 @@ namespace MA5
             closest_approach_.clear();
             d0_         = 0.;
             dz_         = 0.;
+            momentum_rotation_ = 0.;
```

Comment:
I don't understand. This should be there, shouldn't it?

Reply:
> I don't understand. This should be there, shouldn't it?


Yes yes it should I just mean that this should be done in the `main` branch because that bug fix should be released sooner than the `substructure` branch.

Reply:
Ah ok. Sure. Please create a dedicated PR for the main branch and I will approve it asap.

Reply:
It turns out it's already updated in the `main` branch. Once you merge `sharedlib` to `substructure`, I'll merge `main` into this branch so that everything can be up to date.